src/Storage/Controllers/InstancesController.cs

```diff
@@ -276,6 +276,35 @@ public async Task<ActionResult<Instance>> Get(int instanceOwnerPartyId, Guid ins
             }
         }
 
+        /// <summary>
+        /// Gets process info relevant to authorization
+        /// </summary>
+        /// <param name="instanceOwnerPartyId">The party id of the instance owner.</param>
+        /// <param name="instanceGuid">The id of the instance to retrieve.</param>
+        /// <returns>Authorization info.</returns>
+        [HttpGet("AuthInfo/{instanceOwnerPartyId:int}/{instanceGuid:guid}")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        [Produces("application/json")]
+        [ApiExplorerSettings(IgnoreApi = true)]
+        public async Task<ActionResult<AuthInfo>> GetForAuth(int instanceOwnerPartyId, Guid instanceGuid)
```

Comment:
We'll need to add the new type in a separate PR, merge and publish a new version of the package.

Reply:
https://github.com/Altinn/altinn-storage/pull/233

Reply:
We've now published version 3.23.0